src/Sylius/Bundle/ShopBundle/Resources/views/error404.html.twig

```diff
@@ -0,0 +1,13 @@
+{% extends '@SyliusShop/layout.html.twig' %}
+
+{% block content %}
+    <h2 class="ui center aligned icon header">
+        <img src="{{ asset('assets/shop/img/logo.png') }}" alt="Sylius logo" class="ui small image" />
+        <br/><br/>
+        {{ 'sylius.ui.page_not_found.header'|trans }}
```

Comment:
Why this new convention for translation keys ... ?

``sylius.ui.not_found`` will be enough. As for text, I'd suggest something a bit more neutral: ``sylius.ui.the_page_you_are_looking_for_does_not_exist`` and that's it.